client/src/components/Earn/EarnTipTiles/EarnTipTiles.tsx

```diff
@@ -66,7 +66,7 @@ const EarnTipTiles = (): ReactElement => {
   const isAllocateRewardsTipVisible =
     (!wasAllocateRewardsAlreadyClosed &&
       isDecisionWindowOpen &&
-      individualReward !== 0n &&
+      !individualReward &&
```

Comment:
`isAllocateRewardsTipVisible` when the user doesn't have `individualReward`? Why?

Reply:
`master` code for that: `!individualReward?.isZero()`.